BitwardenShared/Core/Platform/Utilities/IncomingLinkConstants.swift

```diff
@@ -0,0 +1,14 @@
+// MARK: - IncomingLinksConstants
+
+/// Links that are coming into our app via the "bitwarden://" deep link scheme.
+///
+enum IncomingLinkConstants {
```

Comment:
🎨 If this is only for `bitwarden://` I would rename this enum because a different scheme could also be an "incoming link". Something like:
```suggestion
enum BitwardenDeepLinkConstants {
```

Reply:
I was trying to differentiate against external links that might involve Bitwarden. But I think having deeplink in there (plus my doc comment) is probably explanatory enough. 👍  Adopted in latest commit.